01.dynamic-programming/Dynamic_Programming.ipynb

```diff
@@ -69,7 +69,7 @@
    "cell_type": "markdown",
```

Comment:
<p>See if math renders properly in comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/test-v1/DeepReinforcementLearningNanodegree/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Reply shouldn't mess up math</p>